promql/promqltest/testdata/functions.test

```diff
@@ -271,15 +272,19 @@ eval instant at 50m deriv(http_requests_total{group="canary", instance="1", job=
 eval instant at 50m deriv(testcounter_reset_middle_total[100m])
 	{} 0.010606060606060607
 
-# deriv should ignore histograms.
-eval instant at 110m deriv(http_requests_mix{group="canary", instance="1", job="app-server"}[110m])
+# deriv should ignore histograms with info annotation.
```

Comment:
This comment sounds like there should be an info annotation for any ignored histogram. However, we only get the annotation for a mix of floats and histograms, while pure histogram ranges are ignored silently. The comment should reflect this. (It could refer to the mix here, and then there could be a comment before line 282 saying that pure histogram ranges are ignored without annotation.)